### PR TITLE
[Add] added identities for SPE CDLC: US, French, German

### LINKED ISSUE #185: [Feature Request] Add SPE identity presets
**Is your feature request related to a problem? Please describe.**
There are no SPE identity presets in the framework right now.

**Describe the solution you'd like**
For there to be presets.

**Why is this this best way to solve the problem?**
I will make it.

**Additional context**
No.

### PR DESCRIPTION
**Pull request by Joecuronium**
### Pull Request Description
**When merged this pull request will:**
add Identity support for the Spearhead CDLC in the same way support was added for GM and SOG
resolves #185 

### Release Notes
added identities for SPE CDLC

## IMPORTANT

- [x] Testing has been completed as neccessary, depending on the nature & impact of the changes.
- [x] The Release Notes section below must be filled out appropriately to explain the changes made in this PR. This section will be used in Framework Changelogs.
- [x] If the contribution affects [the wiki](https://github.com/CombinedArmsGaming/CAFE3/wiki), please include your changes in this pull request so the wiki is consistently updated.
- [x] [Contribution Guidelines](https://github.com/CombinedArmsGaming/CAFE3/blob/release/contributing.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `[Descriptor] - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by Joecuronium** [comment]
Changes are deployed on tonights mish as a live testing environment, multiplayer test on my own did not cause any issues.

File: configuration/identities/identityGroups.sqf [hunk 1]
```diff
@@ -172,3 +172,25 @@ _names = "gm_names_dan_male";
 
 CREATE_IDENTITY(gm_danish_camo,_speakers,_faces,_names);
 
+// [Spearhead CDLC] US
+_speakers = ["Sturrock","WhiteHead_20","WhiteHead_16","WhiteHead_02","GreekHead_A3_07","WhiteHead_08","GreekHead_A3_06","WhiteHead_03","WhiteHead_07","Barklem","WhiteHead_17","WhiteHead_12","WhiteHead_18","WhiteHead_09","WhiteHead_10","WhiteHead_11","WhiteHead_15","GreekHead_A3_08","GreekHead_A3_09","GreekHead_A3_05","AfricanHead_02","AfricanHead_03","WhiteHead_14","WhiteHead_21","WhiteHead_19","WhiteHead_06","WhiteHead_01","WhiteHead_04","WhiteHead_05","AfricanHead_01","WhiteHead_13"];
```
Comment (Bubbus) [inline configuration/identities/identityGroups.sqf]: Speakers array is full of faces?  This one might need changing.

**Review by Bubbus — CHANGES_REQUESTED** [review]

**Comment by Joecuronium** [comment]
fixed now, was intended ;P

**Review by Bubbus — APPROVED** [review]
delightfully devilish joemour

**Pull request merged**